installation/resources/installer-config-local.yaml.tpl

```diff
@@ -73,6 +73,8 @@ data:
 
   mixer.policy.autoscaleEnabled: "false"
   mixer.telemetry.autoscaleEnabled: "false"
+  
+  global.tracer.zipkin.address: zipkin.kyma-system:9411
 ---
```

Comment:
Zipkin address can be moved to values.yaml in the charts directly and removed from config files